Example/ProfileViewController.swift

```diff
@@ -19,7 +19,7 @@ class ProfileViewController: UIViewController {
         guard let familyName = familyNameTextField.text else { return }
         let profile = UserProfile(givenName: givenName, familyName: familyName)
 
-        UIApplication.identityManager.currentUser.profile.update(profile) { result in
+        UIApplication.currentUser.profile.update(profile) { result in
```

Comment:
Looks a bit weird that you are updating `profile` with a new `profile`. Maybe its just a naming issue, but it looks like you could do `UIApplication.currentUser.profile = profile`

Reply:
Yeah, I think could've been better named. If you make it a computed setter then you need async await functionality to get async results :p